packages/core/src/serverless/provider.ts

```diff
@@ -0,0 +1,16 @@
+import { graphql, GraphQLArgs } from 'graphql';
+import generateSchema from '../generators/schema';
+import { FlatbreadConfig } from 'flatbread';
+
+/**
+ * Generates a GraphQL schema and returns a query function that can be used to execute a GraphQL query.
+ *
+ * @param config Flatbread config
+ */
+export async function initServerlessFlatbreadProvider(config: FlatbreadConfig) {
+  const schema = await generateSchema({ config });
+
+  return async function query(args: Omit<GraphQLArgs, 'schema'>) {
+    return graphql({ schema, ...args });
+  };
+}
```

Comment:
@odama626 I moved this here as to prevent a circular dependency between `flatbread` and `@flatbread/core`. 

We could potentially move it out into its own package if we decide that would make more sense.

Also what do you think of the name I changed the function to?

Reply:
I think the name is a little wordy, I think maybe this should be the default functionality since it doesn't require extra libraries like the server does, maybe we should make this the default export and export { getSchema } as a named export, and maybe we could move the server provider to another package? maybe `@flatbread/provider-apollo`?

Reply:
Great suggestion, I agree with those points. It'll be great to break out from anything we impose on in the stack - that'll allow us to fill in the gaps in the ecosystem with custom-fit providers

One aspect to bear in mind is how we do the `flatbread` cli, which is currently bundled with the Apollo provider